airbyte-webapp/src/pages/ConnectionPage/pages/CreationFormPage/components/SourceForm.tsx

```diff
@@ -1,6 +1,6 @@
+// TODO: create separate component for source and destinations forms
```

Comment:
Should this just be tracked in an issue?

Reply:
This isn't from me, looks like import sorting did it. The comment is 17 months old, I dunno if we should even bother holding on to it.

Reply:
Oh! I thought it was a new TODO